js/styling.js

```diff
@@ -50,7 +50,9 @@ jQuery( function ( $ ) {
 			$( window ).trigger( 'panelsStretchRows' );
 		}
 	}
-	$( window ).resize( stretchFullWidthRows ).load( stretchFullWidthRows );
+	$( window ).resize( stretchFullWidthRows ).on('load',function(){
```

Comment:
No need to use a closure here. It's fine to just call `stretchFullWidthRows` directly.
In fact, this could be shortened to just `$( window ).on( 'resize load', stretchFullWidthRows );`

Reply:
@braamgenis initially I wrote it like that but Codacy gave an error "on check" and I changed it to the way codacy asked to satisfy the check.
I'll shorten the code and commit again.
thanks.

Reply:
Well Codacy seems to be happy with it now. :+1: